@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -50,13 +54,14 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
         url.searchParams.set('session_id', xapi.sessionId)
 
         // eslint-disable-next-line no-new
-        new RFB(this.state.container.current, url, {
+        const rfb = new RFB(this.state.container.current, url, {
           wsProtocols: ['binary'],
         })
+        rfb.scaleViewport = true
       },
     },
   },
-  ({ state }) => <div ref={state.container} />
+  ({ state }) => <div ref={state.container} style={state.consoleSize} />
```

Comment:
I think it would be better to pass a `scale` prop instead. Also, state inheritance may be removed from Reaclette in the future.

Reply:
`effects` inheritance too ?

Reply:
Yes.